cypress/e2e/saveModelControl/save-model.cy.js

```diff
@@ -155,7 +163,7 @@ describe('save model', () => {
         cy.wait('@authorizeRequest').its('response.statusCode').should('eq', STATUS_OK)
         cy.wait('@tokenRequest').its('response.statusCode').should('eq', STATUS_OK)
         // Commented out for now
-        // cy.findByTestId('Save IFC', {timeout: 60000}).should('exist')
+        cy.findByTestId('Save IFC', {timeout: 60000}).should('exist')
```

Comment:
I feel like these should be short, e.g. 5s.  If we don't have the popup almost immediately, it's bc it's not going to work.  Or that's how it's been for me with the dialogs i was testing recently

if so, the length for these timeouts should be smth like "to make tests that are going to fail, fail faster, but no so short that we get flaky behavior"

Reply:
PTAL